test/parallel/test-buffer-fill.js

```diff
@@ -249,8 +249,8 @@ function writeToFill(string, offset, end, encoding) {
 
   // Correction for UCS2 operations.
   if (os.endianness() === 'BE' && encoding === 'ucs2') {
-    for (var i = 0; i < buf2.length; i += 2) {
-      var tmp = buf2[i];
+    for (let i = 0; i < buf2.length; i += 2) {
```

Comment:
That rule was created by @TheAlphaNerd to help people avoid the whole "using `let` in `for` loops is slow" performance problem. I would avoid it.